data/json/items/melee.json

```diff
@@ -4066,5 +4066,73 @@
         "bashing": 1,
         "cutting": 0,
         "to_hit": -2
-    }
+    },{
+    "type": "GENERIC",
+    "id": "sawblade_axe",
+    "name": "sawblade axe",
+    "description": "This is an axe made by attaching the blade of a circular saw to a pipe. It makes a decent melee weapon.",
+    "weight": 1600,
+    "to_hit": 1,
```

Comment:
By adding the circular saw blade onto a pipe, it would change from an attack-anywhere to line-of-damage weapon (-1), as well as throwing off the balance (-1). This weapon should have -1 to hit rather than +1 to hit.